x-pack/plugins/endpoint/server/routes/alerts/list/lib.ts

```diff
@@ -3,22 +3,69 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import { SearchResponse } from 'elasticsearch';
 import { KibanaRequest } from 'kibana/server';
 import {
   JsonObject,
   fromKueryExpression,
   toElasticsearchQuery,
-} from '../../../../../../src/plugins/data/common/es_query/kuery';
-import { EndpointAppConstants } from '../../../common/types';
+} from '../../../../../../../src/plugins/data/common/es_query/kuery';
 import {
-  EndpointAppContext,
-  AlertRequestParams,
-  AlertRequestData,
-  AlertRequest,
-  AlertRequestBody,
-} from '../../types';
-
-function buildQuery(reqData: AlertRequestData): JsonObject {
+  EndpointAppConstants,
+  AlertData,
+  AlertDataWrapper,
+  AlertHits,
+  AlertResultList,
+  ESTotal,
+} from '../../../../common/types';
+import { EndpointAppContext } from '../../../types';
+import {
+  AlertListRequestParams,
+  AlertListRequestData,
+  AlertListRequest,
+  AlertListRequestBody,
+} from './types';
+
+export const getRequestData = async (
+  request: KibanaRequest<unknown, AlertListRequestParams, AlertListRequestParams>,
+  endpointAppContext: EndpointAppContext
+): Promise<AlertListRequestData> => {
+  const config = await endpointAppContext.config();
+  const reqData = {} as AlertListRequestData;
```

Comment:
The appears to be requestParams or requestInfo or requestProps? Data denotes something that you are fetching, in my view. What do you think?

Reply:
Yeah, good point. I'm leaning towards `requestQuery` or `reqQuery` ...